### DISCUSSION TITLE
Jquery - swipebox - JS - css - Problems with pagination

### DISCUSSION THREAD

**Discussion by Pikini** [community/community#108437] [Programming Help]
### Body

Goodnight,

I made a responsive website, it turned out very good.

My problem is with the cell phone.

When I click on an image with my finger in the photo gallery, it opens completely on the screen, but when I zoom in to look at the details it goes to the next photo or previous photo.

How do I not go to the next or previous photos with my finger?

I want the previous or next functions only through the buttons.

When I open the photo, I want it locked to look and zoom with my finger. I don't want it to be sensitive going or returning to the next photos.

How do I make this change in the code?

Would it be through my "swipebox.css" file or the "jquery.swipebox.min.js" file??

Thank you very much for the help.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Zain-Muhammad** [comment]
Have you fixed it or not if not let me know i'll help you to fix it.
